p5/core/primitives.py

```diff
@@ -25,7 +25,7 @@
 from ..pmath import curves
 from ..pmath.utils import SINCOS
 
-from .shape import PShape
+from p5.sketch.Vispy2DRenderer.shape import PShape
```

Comment:
We dont have to use the entire namespace. We can use `from ..sketch.Vispy2DRenderer.shape import PShape`

Reply:
These changes in namespaces were done automatically by **pycharm** when I moved (refactored) the files. Should I undo this manually ?

Reply:
Fix these imports. Then I will merge this PR